app/api/endpoints/transfer_request_collections.rb

```diff
@@ -1,7 +1,10 @@
 module Endpoints
   class TransferRequestCollections < ::Core::Endpoint::Base
     model do
-      action(:create, to: :standard_create!)
+      # action(:create, to: :standard_create!)
+      action(:create) do |request, response|
+        standard_create!(request, response)
+      end
```

Comment:
What difference does this change make? (Sorry for all the q's, not familiar with this API)

Reply:
I think no difference, we were changing some parts of code and i think we can roll back this to how it was.